drivers/kw41zrf/kw41zrf_netdev.c

```diff
@@ -1175,9 +1174,9 @@ static uint32_t _isr_event_seq_ccca(kw41zrf_t *dev, uint32_t irqsts)
 static void kw41zrf_netdev_isr(netdev_t *netdev)
 {
     kw41zrf_t *dev = (kw41zrf_t *)netdev;
-    num_irqs_handled = num_irqs_queued;
 
-    if (!spinning_for_irq) {
+    irq_is_queued = false;
+    if (!blocking_for_irq) {
         thread_flags_clear(KW41ZRF_THREAD_FLAG_ISR);
```

Comment:
So `blocking_for_irq` is set when this function gets called from `kw41zrf_wait_idle()` - when are those flags then cleared?

Reply:
After looking at it I don't think that `if` should even be there. Maybe it used to be needed but now I think it's a bug so I removed it. That leaves `blocking_for_irq` being used only for an assertion but I didn't want to remove that one.

Reply:
I'm not seeing those spurious `gnrc_netif: possibly lost interrupt` anymore - nice!